src/js/actions/superselect.js

```diff
@@ -449,6 +449,7 @@ define(function (require, exports) {
     click.action = {
         reads: [locks.PS_DOC, locks.JS_DOC, locks.JS_TOOL, locks.JS_UI],
         writes: [],
+        modal: true,
```

Comment:
No, they transfer to actions that write to the doc state, which is why they don't have write locks. Marking them as modal delays canceling the modal state until the transfer, if that actually occurs.